porting-cheat-sheet.md

```diff
@@ -31,6 +31,12 @@ For .NET Framework projects it may be necessary to include the following in your
 </configuration>
 ```
 
+## Functionality Changes
+
+| System.Data.SqlClient | Microsoft.Data.SqlClient |
+|--|--|
+| Can use DateTime object as value for SqlParameter with type `DbType.Time`. | Must use TimeSpan object as value for SqlParameter with type `DbType.Time`. |
```

Comment:
Also applies to `DbType.Date`, please add that too.
ref: https://github.com/dotnet/SqlClient/pull/269

Reply:
I think the `DbType.Date` change is little less straightforward since there's no easy equivalent of "Use TimeSpan instead" for dates. But hopefully what I've added makes sense.